src/modules/aws_dynamo/entity/index.ts

```diff
@@ -3,59 +3,118 @@ import { Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 't
 import { cloudId } from '../../../services/cloud-id';
 import { AwsRegions } from '../../aws_account/entity';
 
+/**
+ * @enum
+ * Types of tables supported by Dynamo DB
+ * @see https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.TableClasses.html
+ */
 export enum TableClass {
   Standard = 'STANDARD',
   StandardInfrequentAccess = 'STANDARD_INFREQUENT_ACCESS',
 }
 
+/**
+ * Table to manage AWS Dynamo DB tables. Amazon DynamoDB is a fully managed, serverless, key-value
+ * NoSQL database designed to run high-performance applications at any scale. DynamoDB offers built-in security,
+ * continuous backups, automated multi-Region replication, in-memory caching, and data import and export tools.
+ *
+ * @example
+ * ```sql TheButton[Manage a DynamoDB table]="Manage a DynamoDB table"
+ * INSERT INTO dynamo_table (table_name, table_class, throughput, primary_key)
+ * VALUES ('dynamo-table', 'STANDARD','"PAY_PER_REQUEST"', '{"key": "S", "val": "S"}');
+ *
+ * SELECT * FROM dynamo_table  WHERE table_name = 'dynamo-table';
+ *
+ * DELETE FROM dynamo_table WHERE table_name = 'dynamo-table';
+ * ```
+ *
+ * @see https://github.com/iasql/iasql-engine/blob/main/test/modules/aws-dynamo-integration.ts#L90
+ * @see https://aws.amazon.com/dynamodb/
+ *
+ */
 @Entity()
 export class DynamoTable {
+  /**
+   * @private
+   * Auto-incremented ID field for storing builds
+   */
   @PrimaryGeneratedColumn()
   id?: number;
 
+  /**
+   * @public
+   * Name for the Dynamo table
+   */
   @Column({
     /* unique: true, */ // Waiting on Postgres 15 to revive this
     nullable: false,
   })
   @cloudId
   tableName: string;
 
+  /**
+   * @public
+   * Class for the table
+   */
   @Column({
     type: 'enum',
     nullable: false,
     enum: TableClass,
   })
   tableClass: TableClass;
 
-  // TODO: How to constrain this more appropriately in the database?
+  /**
+   * @public
+   * Complex type to represent the provisioned throughput settings for the table
+   * TODO: How to constrain this more appropriately in the database?
```

Comment:
so this TODO shows up in the docs?

Reply:
ok i found that there is a "privateRemarks" annotation that we can use to add internal comments. I included all the TODOS behind that annotation